pkg/webhook/controlplane/ensurer.go

```diff
@@ -264,13 +264,18 @@ func (e *ensurer) EnsureAdditionalUnits(ctx context.Context, ectx genericmutator
 		command              = "start"
 		trueVar              = true
 		customMTUUnitContent = `[Unit]
-    Description=Apply a custom MTU to eth0
-    Requires=eth0.network
+Description=Apply a custom MTU to network interfaces
+After=network-online.target
```

Comment:
Is `network-online.target` present everywhere?

Reply:
isn't this somehow  tied to `NetworkManager` being on the system or not?

Reply:
`network-online.target` is independent from implementation details like NetworkManger.

I've no strong opinion whether it should be `network.target` or `network-online.target`, see details about them here https://wiki.freedesktop.org/www/Software/systemd/NetworkTarget/#runningservicesafterthenetworkisup